openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -181,24 +184,20 @@ public void pluginInit(StfCoreExtension test) throws Exception {
 			throw new StfException(testExecutionType + "Cannot locate the JCK. It must either be placed in a jck subdirectory within a -systemtest-prereqs directory (e.g. /home/user/systemtest_prereqs/jck/jck8b) or be provided via the -test-args=\"jckRoot=xxxx\" option. The check failed because " + jckRoot + " does not point to a directory containing a JCK-runtime directory");
 		}
 		testSuite = testArgs.decodeEnum("testsuite", TestSuite.class);
-		
 		testExecutionType = testArgs.get("executiontype");
 		withAgent = testArgs.get("withagent");
 		interactive = testArgs.get("interactive");
 		concurrencyString = testArgs.get("concurrency");
-		
-		versionNo = jckVersion.replace("jck", "");
-		testSuiteFolder = "JCK-" + testSuite.toString().toLowerCase() + "-" + versionNo;
+		testSuiteFolder = "JCK-" + testSuite.toString().toLowerCase() + "-" + jckVersionNo;
 		platform = test.env().getOsgiOperatingSystemName();	
-		
-		jckBase = test.env().findPrereqDirectory(jckTopDir + jckVersion + "/" + testSuiteFolder);
-		
-		nativesLoc = test.env().findPrereqDirectory(jckTopDir + jckVersion + "/natives/" + test.env().getPlatform());
+		jckBase = test.env().findPrereqDirectory(testSuiteFolder);
+		nativesLoc = test.env().findPrereqDirectory("natives/" + test.env().getPlatform());
 
 		DirectoryRef repositoryConfigLoc = test.env().findTestDirectory("openjdk.test.jck/config");
 		jtiFile = repositoryConfigLoc.childFile("/" + jckVersion + "/" + testSuite.toString().toLowerCase() + ".jti");
+		//fileUrl = "file:///" + jckRoot + "/" + testSuiteFolder + "/testsuite.jtt";
 		
-		fileUrl = "file:///" + test.env().findPrereqFile(jckTopDir + jckVersion + "/" + testSuiteFolder + "/testsuite.jtt");
+		fileUrl = "file:///" + test.env().findPrereqFile(testSuiteFolder + "/testsuite.jtt");
```

Comment:
Should we use `jckRoot` here?

Reply:
When JckRoot is provided as argument to Jck.java, which is the case for us here, jckRoot == jckTopDir.